### PR TITLE
feat(metrics): expose span self time gauge

### PR DESCRIPTION
**Pull request by obostjancic**
Exposes span self time gauge

image

### CODE REVIEW

**Review by iambriccardo — APPROVED** [review]

File: static/app/utils/metrics/mri.spec.tsx [hunk 1]
```diff
@@ -84,6 +84,7 @@ describe('parseMRI', () => {
     ['d:transactions/duration@millisecond', 'transaction.duration'],
     ['d:spans/duration@millisecond', 'span.duration'],
     ['d:spans/exclusive_time@millisecond', 'span.self_time'],
+    ['g:spans/self_time@millisecond', 'span.self_time'],
```
Comment (ArthurKnaus) [inline static/app/utils/metrics/mri.spec.tsx:87]: WiIl we get duplicated metric entries somewhere? 🤔

**Review by ArthurKnaus — APPROVED** [review]

File: static/app/utils/metrics/mri.spec.tsx (hunk 1 above)
Comment (obostjancic) [reply static/app/utils/metrics/mri.spec.tsx:87]: Yes, but i don't know how else to call it 😂

Comment (ArthurKnaus) [reply static/app/utils/metrics/mri.spec.tsx:87]: Hmm, seems sub-optimal to me...

Comment (obostjancic) [reply static/app/utils/metrics/mri.spec.tsx]: moved to `self_time` and `exclusive time`

**Pull request merged**